puzzlegenerator/puzzlegenerator_service/pom.xml

```diff
@@ -19,6 +19,11 @@
 
 	<dependencies>
 		<dependency>
+			<groupId>com.flurnamenpuzzle.generator</groupId>
+			<artifactId>puzzlegenerator_ui</artifactId>
```

Comment:
Habe gerade bemerkt, dass unsere dies ja auch in unserer Package-Struktur so ist. Bleiben wir bei com oder wechseln wir noch auf ch?